doc/adr/008_Client_Side_Rewrite.md

```diff
@@ -0,0 +1,50 @@
+# Doing a Rewrite of the Client-Side Code
+
+Date: 2019-04-25
+
+## Status
+
+Accepted
+
+## Context
+
+The mobile-friendly, more aesthetically pleasing design of the software (called `cat17`) did not gather as much donations as the "old" design, called `10h16`. UX engineering, product management and stakeholders from the fundraising department decided against going "back" to the mobile-unfriendly and hard-to-maintain `10h16` design and instead improving the existing `cat17` design by applying UX best practices and making educated guesses at what made `10h16` so attractive to donors.
+
+The engineering team, planning the modernization of the frontend code base as outlined in "[ADR 006 - Using Vue.js](006_Vue.js.md) and "[ADR 007 - Using Typescript](007_Typescript.md) now had to choose between an *evolution* of `cat17` and a *rewrite* of the client-side code. The benefits of one approach would be the drawbacks of the other, so the following sections will describe each approach through the lens of benefits.
+
+### Benefits of an evolution of `cat17`
```

Comment:
- We don't risk breaking something that already works (ui/ux wise).
- We can improve the skin "step by step" so to speak. First implement the Vue components, then get rid of bootstrap and replace it with bulma possibly, then transform our css structure into a more meaningful one.
- This approach doesn't stop us from developing new features if need be.